src/Sarif/ConsoleLogger.cs

```diff
@@ -263,12 +264,12 @@ private string FormatNotificationMessage(Notification notification)
 
             if (!string.IsNullOrEmpty(notification.Id))
             {
-                sb.Append($" {notification.Id}: ");
+                sb.AppendFormat(CultureInfo.CurrentCulture, notification.Id + ": ");
```

Comment:
I think I commented on a previous version of this: you don't need to "format" at all here. You can just write

```
sb.Append(notification.Id);
```
... and avoid the whole issue of "culture".